src/Plugin/Validation/Constraint/LowercaseEmailConstraintValidator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\apigee_edge\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+/**
+ * Validates the LowercaseEmail constraint.
+ */
+class LowercaseEmailConstraintValidator extends ConstraintValidator {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validate($value, Constraint $constraint) {
+    foreach ($value as $item) {
+      // First check if the organization is ApigeeX.
+      try {
+        /** @var \Drupal\apigee_edge\SDKConnectorInterface $sdk_connector */
+        $sdk_connector = \Drupal::service('apigee_edge.sdk_connector');
+        $org_controller = \Drupal::service('apigee_edge.controller.organization');
+        /** @var \Apigee\Edge\Api\Management\Entity\Organization $organization */
+        $organization = $org_controller->load($sdk_connector->getOrganization());
+
+        // Check if organization is ApigeeX.
+        if ($organization && ('CLOUD' === $organization->getRuntimeType() || 'HYBRID' === $organization->getRuntimeType())) {
```

Comment:
```suggestion
        if ($controller->isOrganizationApigeeX()) {
```

https://github.com/apigee/apigee-edge-drupal/blob/0f0e59aff7cc680dd9ec49825728af94df904b84/src/Entity/Controller/OrganizationController.php#L151

This also makes the whole try-catch dead code, if I am not mistaken.

Reply:
Updated the code. Thanks for the suggestion.